jujugui/static/gui/src/app/assets/javascripts/jujulib/test-jujulib-charmstore.js

```diff
@@ -317,19 +317,26 @@ describe('jujulib charmstore', function() {
     });
 
     it('accepts an author & calls to generate an api path', function() {
-      charmstore.list('test-author');
+      charmstore.list('test-author', 'cb');
       assert.equal(generatePath.callCount, 1, 'generatePath not called');
       assert.deepEqual(generatePath.lastCall.args, [
         'list',
-        'owner=' + 'test-author&' +
+        'owner=test-author&' +
+            'type=charm&' +
             'include=charm-metadata&' +
             'include=bundle-metadata&' +
             'include=bundle-unit-count&' +
-            'include=extra-info%2Fbzr-owner&' +
+            'include=extra-info&' +
             'include=supported-series&' +
             'include=stats']);
     });
 
+    it('can list bundles', function() {
+      charmstore.list('test-author', 'cb', {type: 'bundle'});
+      var qs = generatePath.lastCall.args[1];
+      assert.ok(qs.indexOf('type=bundle'), 'bundle not set in query string');
```

Comment:
`ok` returns truthy for `-1` so you'll want to use equals and check that it's greater than -1